src/test/java/org/blfrias/fakebank/service/CustomerServiceImplTest.java

```diff
@@ -58,4 +58,24 @@ void testGetCustomerById() {
         assertEquals(customerDTO, retrievedCustomer.orElse(null));
         verify(customerRepository, times(1)).findById(any());
     }
+
+    @Test
+    void testDeleteCustomer() {
+        CustomerDTO customerDTO = new CustomerDTO();
+        Customer customer = new Customer();
+        when(customerMapper.toDto(any())).thenReturn(customerDTO);
+        when(customerRepository.findById(any())).thenReturn(Optional.of(customer));
+        customerService.deleteById("123");
+        verify(customerRepository, times(1)).deleteById(any());
+    }
+
+    @Test
+    void testUpdateCustomer() {
+        CustomerDTO customerDTO = new CustomerDTO();
+        Customer customer = new Customer();
+        when(customerMapper.toDto(any())).thenReturn(customerDTO);
+        when(customerMapper.toEntity(any())).thenReturn(customer);
+        when(customerRepository.save(any())).thenReturn(customer);
+
```

Comment:
mag add ng assertions dito to check kung ung customer details are the same sa DTO nung na-save